test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -6,348 +6,358 @@ const {
-          await driver.switchToWindow(windowHandles.popup);
-          await driver.clickElement({ text: 'Edit', tag: 'button' });
-          const inputs = await driver.findElements('input[type="number"]');
-          const gasLimitInput = inputs[0];
-          const gasPriceInput = inputs[1];
-          await gasLimitInput.fill('4700000');
-          await gasPriceInput.fill('20');
-          await driver.waitForSelector({ text: 'Save', tag: 'button' });
-          await driver.clickElement({ text: 'Save', tag: 'button' });
-          await driver.clickElement({ text: 'Confirm', tag: 'button' });
-
-          await driver.switchToWindow(windowHandles.dapp);
-
-          const tokenContractAddress = await driver.waitForSelector({
-            css: '#tokenAddress',
-            text: '0x',
-          });
-          const tokenAddress = await tokenContractAddress.getText();
-
-          // imports custom token from extension
-          await driver.switchToWindow(windowHandles.extension);
-          await driver.clickElement(`[data-testid="home__asset-tab"]`);
-          await driver.clickElement({ tag: 'button', text: 'Assets' });
-
-          await driver.clickElement({ text: 'import tokens', tag: 'a' });
-          await driver.clickElement({
-            text: 'Custom token',
-            tag: 'button',
-          });
-          await driver.waitForSelector('#custom-address');
-          await driver.fill('#custom-address', tokenAddress);
-          await driver.waitForSelector('#custom-symbol');
-          await driver.waitForSelector('#custom-decimals');
-          await driver.delay(2000);
-
-          await driver.clickElement({
-            text: 'Add custom token',
-            tag: 'button',
-          });
-
-          await driver.delay(2000);
-          await driver.clickElement({
-            text: 'Import tokens',
-            tag: 'button',
-          });
-
-          // renders balance for newly created token
-          await driver.waitForSelector('.app-header__logo-container');
-          await driver.clickElement('.app-header__logo-container');
-          await driver.clickElement({ tag: 'button', text: 'Assets' });
-          const asset = await driver.waitForSelector({
-            css: '.asset-list-item__token-value',
-            text: '10',
-          });
-          assert.equal(await asset.getText(), '10');
-        },
-      );
-    });
+const { SMART_CONTRACTS } = require('../seeder/smart-contracts');
+
+describe('Create token, approve token and approve token without gas', function () {
+  const smartContract = SMART_CONTRACTS.HST;
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+
+  it('creates, imports and renders the balance for the new token', async function () {
```

Comment:
We can also remove `creates` from the test title

Reply:
Done